### DISCUSSION TITLE
How to re-generate thumbnails?

### DISCUSSION THREAD

**Discussion by hoggatt** [photoprism/photoprism#1466] [Q&A] [answered]
Hey there,
I'd like to preface by saying that this is an awesome app! 

So, I decided to shrink my 'max static thumbnail' size and now regret it. How can I regenerate all of the thumbnails after changing the setting back to something a bit larger? I'm guessing/hoping it's just a matter of deleting a couple folders and reindexing but I just want to make sure. 

Thank you!

**Comment by lastzero** [comment, marked as answer]
Run the thumb command in a terminal (see docker-compose.yml inline docs at the top for examples of how to run commands) or reindex, although that may take longer.

**Comment by HeikoGr** [comment]
to all those who were led here by google:
`docker compose exec photoprism photoprism thumbs -f` executed in the photoprism folder.
you must be aware that this will run the command as root.

More infos (and how to change the user) in the docs:
https://docs.photoprism.app/getting-started/docker-compose/

**Reply by AnderssonPeter** [reply]
Thanks for giving back and adding the actual command here!

**Reply by lastzero** [reply]
It's also shown in our docs:
![IMG_2247](https://github.com/photoprism/photoprism/assets/301686/be2cde6f-5ba9-410b-b557-881edf5c6f58)
